libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/RouteMapFragment.kt

```diff
@@ -105,9 +105,11 @@ class RouteMapFragment : CycleMapFragment(), Route.Listener {
         super.onActivityResult(requestCode, resultCode, data)
         if ((requestCode == CIRCULAR_ROUTE_ACTIVITY_REQUEST_CODE) && (resultCode == Activity.RESULT_OK)) {
             if (data != null) {
+                val distance = data.getIntExtra(EXTRA_CIRCULAR_ROUTE_DISTANCE, 0)
+                val duration = data.getIntExtra(EXTRA_CIRCULAR_ROUTE_DURATION, 0)
```

Comment:
How annoying that this only allows primitives, so you have to do the second defaulting below!

Ah well.

Reply:
I'm still getting this error message on my phone, which is on beta version.  When should I expect the changes to take effect?